src/NServiceBus.AcceptanceTests/Core/FakeTransport/FakeQueueCreator.cs

```diff
@@ -1,13 +1,22 @@
 namespace NServiceBus.AcceptanceTests.Core.FakeTransport
 {
+    using System;
     using System.Threading.Tasks;
     using Transport;
 
     class FakeQueueCreator : ICreateQueues
     {
+        public FakeQueueCreator(Action action = null)
```

Comment:
the param doesn't have to be nullable, but I actually like it as it clearer on why we're using the `?.` to invoke the callback.